app/src/main/java/com/portfolio/tasky/entry/models/AuthenticationModel.kt

```diff
@@ -0,0 +1,9 @@
+package com.portfolio.tasky.entry.models
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+data class AuthenticationRequest(@Expose @SerializedName("email")var email : String?, @Expose @SerializedName("password")var password : String?)
+
+data class AuthenticationResponse(@Expose @SerializedName("token")var token : String?, @Expose @SerializedName("userId")var userId : String?,
+                               @Expose @SerializedName("fullName")var fullName : String? )
```

Comment:
I'm not a fan of having this in just one line
I rather have it like:

```
data class AuthenticationRequest(
    @Expose @SerializedName("email") var email : String?, 
    @Expose @SerializedName("password") var password : String?
)
```

It's a bit easier to read

Reply:
Also, you shouldn't need to use @Expose

Since @Expose sets the serialization to true by default. Which if you don't add it, is the default value

So just having @SerializedName is more than enough

Reply:
Single line  - Done

I tried keeping only @SerializedName but then it doesn't work. I have encountered such scenarios before where @Expose becomes necessary to add since it tells the retrofit that the variable can be used for api communication

Reply:
Once I implement Moshi, will get rid of this code.